proteus/mprans/SW2DCV.py

```diff
@@ -356,7 +356,7 @@ def evaluate(self, t, c):
     def preStep(self, t, firstStep=False):
         if firstStep:
             # Init boundaryIndex
-            assert (self.model.boundaryIndex is None and self.model.normalx is not None, "Check boundaryIndex, normalx and normaly")
+            assert self.model.boundaryIndex is None and self.model.normalx is not None, "Check boundaryIndex, normalx and normaly"
```

Comment:
didn't @ejtovar observe the opposite behavior being necessary for python3?

Reply:
I thought he saw the same. The conda build  of python seems to do a better job of explaining why, but basically assert is a keywork, not a function, and no-empty tuples always evaluate to True, so this is a nefarious bug. If you want to do multi-line use \ or be careful to enclose only the test in (): 

https://stackoverflow.com/questions/3112171/python-assert-with-and-without-parenthesis

Reply:
Okay I see. I've seen a lot of `raise(NameError, 'string')` statements in setup files so it looked like the parentheses were supposed to be there.